omod/src/main/webapp/resources/htmlforms/oncologyScreening.html

```diff
@@ -188,42 +188,44 @@
                                 <th>Findings</th>
                                 <th>Treatment/Action</th>
                             </tr>
-                        <includeIf velocityTest="$patient.gender == 'F' ">
-                            <tr>
-                                <th rowspan="3">Cervical</th>
-                                <th>HPV (Age>30 years)</th>
-                                <td><obs conceptId="1169AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
-                                         answerConceptIds="703AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,664AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
-                                         answerLabels="Positive,Negative" style="radio" answerSeparator="" />
-                                </td>
-                                <td><obs conceptId="1169AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
-                                         labelText="&lt;br/&gt;" answerSeparator="&lt;br/&gt;"
-                                         answerConceptIds="703AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,664AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,664AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
-                                         answerLabels="Counseled on Negative result,Referred for VIA/VILI,PAP smear" style="radio" /></td>
-                            </tr>
-                            <tr>
+                            <includeIf velocityTest="$patient.gender == 'F' ">
+                                <tr>
+                                    <th rowspan="3">Cervical</th>
+                                    <includeIf velocityTest="$patient.age >30 ">
```

Comment:
Please check if this > works, it might need to be replaced by &lt; just check

Reply:
works well